persist/boltdb.go

```diff
@@ -26,6 +26,10 @@ type BoltItem struct {
 	Value      []byte
 }
 
+var (
+	ErrNilEntry = errors.New("entry does not exist")
+)
```

Comment:
I believe an identical error is defined in addblock.go as well, so you can replace those errors with this one.